app/models/ability.rb

```diff
@@ -7,11 +7,17 @@ def initialize(user)
     if (user.roles || []).include?("Admin")
       can :manage, :all
     end
+
     # TODO: Refine me
     if (user.roles || []).include?("User")
       can [:create, :read, :update], Ticket, creator_id: user.id
       can :update, Ticket, assignee_id: user.id
     end
+
+    if (user.roles || []).include?("Approver")
```

Comment:
That's not fully correct here... it needs to be:

- User is an Approver
- Ticket needs approval